library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -789,7 +789,7 @@ def adapt_old_config!
     # @param [Hash] a map with netconfig (ifcfg) configuration
     #
     def add_device(device, ifcfg)
-      devtype = GetTypeFromIfcfg(ifcfg) || GetType(device)
+      devtype = GetType(device)
```

Comment:
When I want to understand the change, I look up the names in http://apitalism.herokuapp.com/?q=GetType
and get
- http://www.rubydoc.info/github/yast/yast-yast2/Yast%2FNetworkInterfacesClass%3AGetTypeFromIfcfg
  Detects device type according given ifcfg configuration
- http://www.rubydoc.info/github/yast/yast-yast2/Yast%2FNetworkInterfacesClass%3AGetType 
Detects device type according cached data

  If cached ifcfg for given device is found it is used as parameter for GetTypeFromIfcfgOrName( dev, ifcfg). Otherwise is device handled as unconfigured and result is equal to GetTypeFromIfcfgOrName( dev, nil)

I am still lost. What is the difference in the return value for this particular bug?